questionpy/_ui.py

```diff
@@ -52,21 +71,20 @@ def create_jinja2_environment(attempt: "Attempt", question: "Question") -> jinja
     for package in qpy_env.packages.values():
         loader = _get_loader(package)
         if loader:
-            loader_mapping[f"@{package.manifest.namespace}.{package.manifest.short_name}"] = loader
-
-    # Add a place for SDK-Templates, such as the one used by ComposedAttempt etc.
-    loader_mapping["qpy"] = jinja2.PackageLoader(__package__)
+            loader_mapping[f"@{package.manifest.namespace}/{package.manifest.short_name}"] = loader
 
-    loaders: list[jinja2.BaseLoader] = [jinja2.PrefixLoader(mapping=loader_mapping)]
+    loaders: list[jinja2.BaseLoader] = [_CustomPrefixLoader(mapping=loader_mapping)]
 
     # Get caller package template loader.
-    module_parts = attempt.__module__.split(".", maxsplit=2)
-    if len(module_parts) < 2:  # noqa: PLR2004
+    try:
+        module_parts = attempt.__module__.split(".", maxsplit=2)
+        namespace, short_name, *_ = module_parts
+        key = PackageNamespaceAndShortName(namespace=namespace, short_name=short_name)
+        package = qpy_env.packages[key]
+    except (KeyError, ValueError) as e:
         msg = "Please do not modify the '__module__' attribute."
```

Comment:
Ein Vorschlag für eine aussagekrätigere Fehlermeldung, ist aber nicht so wichtig:
```suggestion
        msg = "Current package namespace and shortname could not be determined from '__module__' attribute. Please do not modify the '__module__' attribute."
```